src/gui/basegraphgl.cpp

```diff
@@ -151,10 +164,26 @@ void BaseGraphGL::slotDeleteSelectedNodes()
     updateCache();
 }
 
+void BaseGraphGL::slotDeleteSelectedEdges()
+{
+    for (auto edge : m_selectedEdges) {
+        m_abstractGraph->removeEdge(edge.second);
+    }
+
+    clearSelection();
```

Comment:
you'll probably get a crash here
have you tested removing an edge?

Reply:
Hm I'm still looking into it. It only crashes when deleting an edge you've added through the graph editor 🤔